### PR TITLE
feat(fab): Add support for increased touch target to mini FAB.

### PR DESCRIPTION
**Pull request by joyzhong**

### CODE REVIEW

File: packages/mdc-fab/_mixins.scss [hunk 1]
```diff
@@ -305,6 +312,17 @@ $mdc-fab-icon-enter-duration_: 180ms;
   }
 }
 
+@mixin mdc-fab--touch_($query: mdc-feature-all()) {
```
Comment (abhiomkar) [inline packages/mdc-fab/_mixins.scss]: Can we move styles from this mixin and apply directly to `.mdc-fab--touch` css selector?

File: packages/mdc-fab/_mixins.scss [hunk 2]
```diff
@@ -305,6 +312,17 @@ $mdc-fab-icon-enter-duration_: 180ms;
...
+  @include mdc-touch-target-component(
```
Comment (abhiomkar) [inline packages/mdc-fab/_mixins.scss]: Not in the scope of this PR but would it make sense to rename `mdc-touch-target-component()` to `mdc-touch-target-margin()`? I think this mixin is only applying margin to increase its touch size.

WDYT?

Comment (joyzhong) [reply packages/mdc-fab/_mixins.scss]: Makes sense! I'll do this in a follow-up CL/PR.

File: packages/mdc-fab/_mixins.scss [hunk 3]
```diff
@@ -25,11 +25,12 @@
...
 @import "@material/ripple/variables";
-@import "@material/theme/functions";
-@import "@material/theme/mixins";
+@import "@material/rtl/mixins";
 @import "@material/shape/mixins";
 @import "@material/shape/functions";
-@import "@material/rtl/mixins";
+@import "@material/theme/functions";
+@import "@material/theme/mixins";
+@import "@material/touch-target/mixins";
```
Comment (abhiomkar) [inline packages/mdc-fab/_mixins.scss]: FAB's package.json missing dependency for touch target which might be causing failures in [canary release](https://github.com/material-components/material-components-web/commit/0c4d8f3923f9a089132ed8dca4062b72d3576aca/checks?check_suite_id=299135665)?

Comment (joyzhong) [reply packages/mdc-fab/_mixins.scss]: Sent https://github.com/material-components/material-components-web/pull/5236 to fix.

**Pull request merged**